applications/FemToDemApplication/custom_elements/generic_small_strain_femdem_element.cpp

```diff
@@ -40,24 +40,28 @@ GenericSmallStrainFemDemElement<TDim, TyieldSurf>::GenericSmallStrainFemDemEleme
 	mThisIntegrationMethod = GetGeometry().GetDefaultIntegrationMethod();
 
 	// Each component == Each edge
-	mNonConvergedThresholds = ZeroVector(NumberOfEdges);   // Equivalent stress
-	mThresholds = ZeroVector(NumberOfEdges); // Stress mThreshold on edge
-	mDamages = ZeroVector(NumberOfEdges); // Converged mDamage on each edge
-	mNonConvergedDamages = ZeroVector(NumberOfEdges); // mDamages on edges of "i" iteration
+    mNonConvergedThresholds.resize(NumberOfEdges);
```

Comment:
First check the size, later resize:

~~~cpp
if (mNonConvergedThresholds.size() != NumberOfEdges)
	mNonConvergedThresholds.resize(NumberOfEdges, false);
~~~